packages/web-react/src/components/Tooltip/README.md

```diff
@@ -0,0 +1,231 @@
+# Tooltip
+
+## Usage
+
+### Basic
+
+```javascript
+<TooltipWrapper>
+  <button type="button" className="Button Button--primary Button--medium TooltipTarget">
+    I have a tooltip!
+  </button>
+  <Tooltip>Hello there!</Tooltip>
+</TooltipWrapper>
+```
+
+### Dismissible
+
+```javascript
+import { useState } from 'react';
+
+const [open, setOpen] = useState(true);
+const closeHandler = () => setOpen(false);
+
+<TooltipWrapper>
+  <button type="button" className="Button Button--primary Button--medium">
```

Comment:
This should be an instance of `Button` component. Or is it intentional?

Reply:
This is not intentional. We have this in multiple places and we should fix it.